### PR TITLE
Added scripts for Use Case 3

### PR DESCRIPTION
**Pull request by melburnerodrigues**
This PR adds the following scripts for Use Case 3
- Terraform scripts
- Pub/Sub publisher program
- Pipeline automation scripts
- Delay (simluated latency) plugin

### CODE REVIEW

File: Usecase 3/delay-plugin/src/main/java/io/delay/plugin/DelayTransform.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
+package io.delay.plugin;
```
Comment (CuriousVini) [inline Usecase 3/delay-plugin/src/main/java/io/delay/plugin/DelayTransform.java:17]: Please add license to javadoc and pom files

File: Usecase 3/delay-plugin/target/classes/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Manifest-Version: 1.0
```
Comment (CuriousVini) [inline Usecase 3/delay-plugin/target/classes/META-INF/MANIFEST.MF]: why are class files and manifest files checked in? Please create gitignore file

File: Usecase 3/delay-plugin/src/main/java/io/delay/plugin/DelayTransform.java (hunk 1 above)
Comment (melburnerodrigues) [reply Usecase 3/delay-plugin/src/main/java/io/delay/plugin/DelayTransform.java:17]: Done

File: Usecase 3/delay-plugin/target/classes/META-INF/MANIFEST.MF (hunk 2 above)
Comment (melburnerodrigues) [reply Usecase 3/delay-plugin/target/classes/META-INF/MANIFEST.MF]: Forgot to add the target directory to the .gitignore file. Apologies. This has been fixed now

**Review by CuriousVini — APPROVED** [review]

**Pull request merged**